exporter/datasetexporter/README.md

```diff
@@ -0,0 +1,62 @@
+# DataSet Exporter
+
+<!-- status autogenerated section -->
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [development]   |
+| Supported pipeline types | logs   |
+| Distributions            | [contrib] |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+This exporter sends logs to [DataSet](https://www.dataset.com/).
+
+See the [Getting Started](https://app.scalyr.com/help/getting-started) guide.
+
+## Configuration
+
+### Required Settings
```

Comment:
It could be worth considering putting the settings into a table, [here's an example](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver#configuration). What do you think?

Reply:
I have randomly clicked on 5 exporters - https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter - and I used the same style that everybody else.

If table is new way, I can update it.

Reply:
I don't think there's an official or recommended way to do it. I like the structure tables offer, which is why I suggested it, but you're free to choose whatever format you prefer; they both offer the same information.